src/Services/Signatures/HashSignature.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace LearnositySdk\Services\Signatures;
+
+use LearnositySdk\Exceptions\ValidationException;
+
+class HashSignature implements SignatureInterface
+{
+    const ALGORITHM = 'sha256';
+
+    const SIGNATURE_VERSION = '01';
+
+    const CONSUMER_KEY_LENGTH = 16;
+
+    const TIMESTAMP_KEY_LENGTH = 13;
+
+    const SIGNATURE_KEY_LENGTH = 64;
+
+    const EXCEPTION_MESSAGE =
+        'The pre hash string for this signature type must contain the secret key';
```

Comment:
Is this correct? The secret has been specifically removed here: https://github.com/Learnosity/learnosity-sdk-php/pull/65/files?diff=unified&w=0#diff-9cdfeaf532c7829f503d5b40728d14527f240f4291868697dfe8ef5687b05e15L282-L284

I think we should reprepend it ourselves here.